lib/catalog/owid/catalog/variables.py

```diff
@@ -303,8 +303,11 @@ def amend_log(
             inplace=inplace,
         )
 
-    def copy_metadata(self, from_variable: "Variable", inplace: bool = False) -> Optional["Variable"]:
-        return copy_metadata(to_variable=self, from_variable=from_variable, inplace=inplace)
+    def copy(self, deep: bool = True) -> "Variable":
```

Comment:
Why are you removing `copy_metadata`? It is useful to have both options at the variable level: `copy` to copy the data (and optionally the metadata), and `copy_metadata` (to copy only the metadata). The latter is useful, for example, after doing an operation that does not propagate metadata, e.g. `tb1["b"] = tb1.groupby("a").transform("sum").copy_metadata(tb1["a"])`

Reply:
My bad, I didn't mean to remove it.